src-frontend/app/controllers/procedures/index.js

```diff
@@ -20,6 +21,15 @@ export default Ember.Controller.extend({
 
             procedure.save().then(function() {
                 procedureController.transitionToRoute('procedure', procedure);
+            }).catch(function(response) {
+                var errorMessage = 'Something went wrong trying to create the procedure';
```

Comment:
I can't call that function from within this controller. @ConnorCimowsky is there some Ember convention for putting JS helper functions (not for Handlebars)